client/src/styles/ConversationPageStyle.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
```

Comment:
DRY, same as in [client/src/styles/ConversationNoData.jsx](https://github.com/Makes-Innovation-Hub/arabrew/pull/54/files#diff-49366bdc0501ae74a888e59e299c815f421034e209a478933141f35ac979b072)

Reply:
There are differents-
in ConversationPageStyle is  justify-content: center;

while in ConversationNoData is:
padding: 1rem;
  padding-bottom: 10rem;